saturn-r/Dockerfile

```
@@ -0,0 +1,21 @@
+
+ARG SATURNBASE_IMAGE
+FROM ${SATURNBASE_IMAGE}
+
+COPY environment.yml /tmp/environment.yml
+COPY postBuild /tmp/postBuild.sh
+COPY r-jupyter-kernel /tmp/r-jupyter-kernel.sh 
+# https://stat.ethz.ch/R-manual/R-devel/library/base/html/libPaths.html
+ENV R_LIBS=/usr/local/lib/R/
+
+RUN conda env update -n saturn --file /tmp/environment.yml && \
+    find ${CONDA_DIR} -type f,l -name '*.pyc' -delete && \
+    find ${CONDA_DIR} -type f,l -name '*.a' -delete && \
+    find ${CONDA_DIR} -type f,l -name '*.js.map' -delete
+    
+RUN /bin/bash -e -u -o pipefail /tmp/postBuild.sh && \
```

```suggestion
    find ${CONDA_DIR} -type f,l -name '*.js.map' -delete && \
    /bin/bash -e -u -o pipefail /tmp/postBuild.sh && \
```

Could these two steps be combined? Using fewer layers will reduce the size of the built mage, maybe by as much as a few MB, and we're building with `--no-cache` anyway so we don't lose anything by combining layers.

---

I don't think I know enough about what the .js.map step is doing to tell, but I trust you!